sample-app.yaml

```diff
@@ -0,0 +1,50 @@
+# event-display app deployment
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: python-ceph-vectordb
+spec:
+  replicas: 1
+  selector:
+    matchLabels: &labels
+      app: python-ceph-vectordb
+  template:
+    metadata:
+      labels: *labels
+    spec:
+      containers:
+        - name: python-ceph-vectordb
+          # Source code: https://github.com/knative/eventing/tree/main/cmd/event_display
+          image: quay.io/jthottan/pythonwebserver:helloworld-python
+          envFrom:
+          - configMapRef:
+              name: ceph-notification-bucket
+          - secretRef:
+              name: ceph-notification-bucket
+          - configMapRef:
+              name: python-ceph-vectordb
+
+---
+# Service that exposes python-vector-db-app app.
+# This will be the subscriber for the Trigger
+kind: Service
+apiVersion: v1
+metadata:
+  name: python-ceph-vectordb
+spec:
+  selector:
+    app: python-ceph-vectordb
+  ports:
+    - protocol: TCP
+      port: 80
+      targetPort: 8080
+---
+#If it is a operator most lively this configmap will be part of CRD values
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: python-ceph-vectordb
```

Comment:
do we need a separate config map per bucket as well?

Reply:
I guess the object type is different here, dimension can be figured from the embedding function. So we some metadata info the bucket itself to know the type object. Then we can rely with one config map for all the settings